tests/test_api.py

```diff
@@ -37,3 +38,29 @@ def test_make_bundle_with_yaml_list(yaml_files, expected):
     with open(expected, "r") as expected_file:
         expected_bundle = yaml.safe_load(expected_file)
         assert unformat_bundle(bundle) == unformat_bundle(expected_bundle)
+
+
+@pytest.mark.parametrize('yaml_files', [
+    ["tests/test_files/bundles/api/bundle1/crd.yml",
+     "tests/test_files/bundles/api/bundle1/csv.yaml"]
+])
+def test_make_bundle_invalid(yaml_files):
+    yamls = []
+    for file in yaml_files:
+        with open(file, "r") as yaml_file:
+            yamls.append(yaml_file.read())
+
+    with pytest.raises(OpCourierBadBundle) as e_info:
```

Comment:
why `e_info` and not just `e` or `exc`?

Reply:
`pytest.raises()` wraps the actual exception in an `ExceptionInfo` object, hence `e_info`. I'm down to change it though if that makes it more readable (tomorrow, that is :smile: )

Reply:
Ah I didn't realize that, it is ok then :D